src/components/TabButton.js

```diff
@@ -0,0 +1,14 @@
+const TabButton = (props) => {
+  const { name, isActive } = props;
+  const className = isActive ? "tab-button active-tab" : "tab-button";
```

Comment:
분기 처리가 필요한 부분으로 좁혀주는 것을 권장드려요.

```suggestion
  const className = `tab-button ${isActive ? "active-tab" : ""}`;
```

Reply:
수정했습니다!

[bcf6294](https://github.com/woowacourse/javascript-lunch/pull/265/commits/bcf62947fe3491e4865d1ed57d7a6332117053f8)